--- a/Modules/GNUInstallDirs.cmake.orig	2021-11-20 19:40:26.652043571 +0100
+++ b/Modules/GNUInstallDirs.cmake	2021-11-20 19:43:12.369286000 +0100
@@ -262,6 +262,8 @@
         set(__system_type_for_install "alpine")
       elseif (EXISTS "/etc/arch-release")
         set(__system_type_for_install "arch linux")
+      elseif (EXISTS "/etc/artix-release")
+        set(__system_type_for_install "artix linux")
       elseif (EXISTS "/etc/debian_version")
         set(__system_type_for_install "debian")
       endif()
