 meson.build | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/meson.build b/meson.build
index cfa01e7a66..738c854e7c 100644
--- a/meson.build
+++ b/meson.build
@@ -3862,11 +3862,11 @@ exe = custom_target(
         install_dir : bindir)
 public_programs += exe

-if want_tests != 'false'
-        test('test-kernel-install',
-             test_kernel_install_sh,
-             args : [exe.full_path(), loaderentry_install])
-endif
+# if want_tests != 'false'
+#         test('test-kernel-install',
+#              test_kernel_install_sh,
+#              args : [exe.full_path(), loaderentry_install])
+# endif

 ############################################################


