--- a/ykpers-args.h     2021-06-06 20:50:51.783331853 +0200
+++ b/ykpers-args.h.patched     2021-06-06 21:05:06.176665036 +0200
@@ -33,8 +33,8 @@
 
 #include "ykpers.h"
 
-const char *usage;
-const char *optstring;
+extern const char *usage;
+extern const char *optstring;
 
 int args_to_config(int argc, char **argv, YKP_CONFIG *cfg, char *oathid,
                   size_t oathid_len, const char **infname,
