From 6b4ebd26a351a05579e9b0cbf330c05f14622079 Mon Sep 17 00:00:00 2001
From: Nathan <ndowens@artixlinux.org>
Date: Thu, 15 Dec 2022 14:41:12 -0600
Subject: [PATCH] libtool-path

---
 autogen.sh | 20 +++++++++-----------
 1 file changed, 9 insertions(+), 11 deletions(-)

diff --git a/autogen.sh b/autogen.sh
index 6579daa..1921838 100755
--- a/autogen.sh
+++ b/autogen.sh
@@ -75,15 +75,13 @@ fi
 rm -f build/config.guess build/config.sub
 $libtoolize --copy --automake --force
 
-ltpath="`dirname $libtoolize`"
-
 if [ "x$LIBTOOL_M4" = "x" ]; then
     ltm4_error='(try setting the LIBTOOL_M4 environment variable)'
-    if [ -d "$ltpath/../share/aclocal/." ]; then
-        ltm4=`cd "$ltpath/../share/aclocal" && pwd`
+    if [ -d "/usr/share/aclocal" ]; then
+        ltm4=`cd "/usr/share/aclocal" && pwd`
     else
         echo "Libtool helper path not found $ltm4_error"
-        echo "  expected at: '$ltpath/../share/aclocal'"
+        echo "  expected at: '/usr/share/aclocal'"
         exit 1
     fi
 else
@@ -125,14 +123,14 @@ done
 if [ $lt_major_version -ge 2 ]; then
     if [ "x$LIBTOOL_CONFIG" = "x" ]; then
         ltconfig_error='(try setting the LIBTOOL_CONFIG environment variable)'
-        if [ -d "$ltpath/../share/libtool/config/." ]; then
-            ltconfig=`cd "$ltpath/../share/libtool/config" && pwd`
-        elif [ -d "$ltpath/../share/libtool/build-aux/." ]; then
-            ltconfig=`cd "$ltpath/../share/libtool/build-aux" && pwd`
+        if [ -d "/usr/share/libtool/config/." ]; then
+            ltconfig=`cd "/usr/share/libtool/config" && pwd`
+        elif [ -d "/usr/share/libtool/build-aux/." ]; then
+            ltconfig=`cd "/usr/share/libtool/build-aux" && pwd`
         else
             echo "Autoconf helper path not found $ltconfig_error"
-            echo "  expected at: '$ltpath/../share/libtool/config'"
-            echo "           or: '$ltpath/../share/libtool/build-aux'"
+            echo "  expected at: '/usr/share/libtool/config'"
+            echo "           or: '/usr/share/libtool/build-aux'"
             exit 1
         fi
     else
-- 
2.39.0

