From 3935c3466b0ebdd11fd2272bb07b8edefb66ae47 Mon Sep 17 00:00:00 2001
From: Nathan <ndowens@artixlinux.org>
Date: Wed, 14 Dec 2022 22:17:52 -0600
Subject: [PATCH] libtool path

---
 autogen.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/autogen.sh b/autogen.sh
index 6579daa..b72df91 100755
--- a/autogen.sh
+++ b/autogen.sh
@@ -64,7 +64,7 @@ done
 #
 
 # Much like APR except we do not prefer libtool 1 over libtool 2.
-libtoolize="`./build/PrintPath glibtoolize libtoolize glibtoolize1 libtoolize15 libtoolize14`"
+libtoolize="`which glibtoolize libtoolize glibtoolize1 libtoolize15 libtoolize14`"
 lt_major_version=`$libtoolize --version 2>/dev/null | sed -e 's/^[^0-9]*//' -e 's/\..*//' -e '/^$/d' -e 1q`
 
 if [ "x$libtoolize" = "x" ]; then
-- 
2.39.0

