From 04e3994ef4666d9a03f61dad35a6788ecc02ffbf Mon Sep 17 00:00:00 2001
From: Nathan <ndowens@artixlinux.org>
Date: Wed, 25 Aug 2021 19:02:16 -0500
Subject: [PATCH] fix format security

---
 exp_clib.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/exp_clib.c b/exp_clib.c
index 1699775..edf153e 100644
--- a/exp_clib.c
+++ b/exp_clib.c
@@ -1476,8 +1476,8 @@ expDiagLogU(str)
      char *str;
 {
   if (exp_is_debugging) {
-    fprintf(stderr,str);
-    if (exp_logfile) fprintf(exp_logfile,str);
+    fprintf(stderr, "%s", str);
+    if (exp_logfile) fprintf(exp_logfile, "%s", str);
   }
 }
 
-- 
2.33.0

