From 3abbb85b5485e2f27983a3b33f17c7faaf4a7e24 Mon Sep 17 00:00:00 2001
From: Dudemanguy <dudemanguy@artixlinux.org>
Date: Wed, 16 Mar 2022 08:44:35 -0500
Subject: [PATCH] fix collections usage for python 3.10

---
 pylib/gyp/common.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/pylib/gyp/common.py b/pylib/gyp/common.py
index b268d229..c80d985a 100644
--- a/pylib/gyp/common.py
+++ b/pylib/gyp/common.py
@@ -4,7 +4,7 @@
 
 from __future__ import with_statement
 
-import collections
+import collections.abc
 import errno
 import filecmp
 import os.path
@@ -494,7 +494,7 @@ def uniquer(seq, idfun=None):
 
 
 # Based on http://code.activestate.com/recipes/576694/.
-class OrderedSet(collections.MutableSet):
+class OrderedSet(collections.abc.MutableSet):
   def __init__(self, iterable=None):
     self.end = end = []
     end += [None, end, end]         # sentinel node for doubly linked list
-- 
2.35.1

